frontend/vue/components/UserAccount/UserAccountLayout.vue

```diff
@@ -59,41 +52,35 @@
 <script lang="ts">
 import { defineComponent } from 'vue-demi'
 import AppLink from '../common/AppLink.vue'
+import BasicLink from '../common/BasicLink.vue'
 import PrivacySection from './PrivacySection.vue'
 import UserProgress from './UserProgress.vue'
 
 export default defineComponent({
   name: 'UserAccountLayout',
   components: {
     AppLink,
+    BasicLink,
     PrivacySection,
     UserProgress
   },
   props: {
     userName: { type: String, required: false, default: '' },
-    userRole: { type: String, required: false, default: '' }
   },
   data () {
     return {
       activeSection: '',
       sectionList: [
         {
-          displayName: this.$translate('My Learning'),
+          displayName: this.$translate('Learning'),
           hash: '#MyLearning'
         },
-        {
-          displayName: this.$translate('Groups'),
-          hash: '#Groups'
-        },
-        {
-          displayName: this.$translate('Badges'),
-          hash: '#Badges'
-        },
         {
           displayName: this.$translate('Privacy'),
           hash: '#Privacy'
         }
-      ]
+      ],
+      logoutRoute: '/logout'
```

Comment:
@vabarbosa - I'll need some guidance here, to figure out  how we tie this in w/ Mathigon's `/logout`. I don't think this is the complete piece, and I'm guessing I'll need to add something to `app.ts`?

Reply:
In mathigon we have a [`logout` end-point](https://github.com/Qiskit/platypus/pull/174/files#diff-6eef9123bed920a90b44258c0f446f14d024a1a2f14b80517f0daf2c6b29ac42R149) that removes the session and redirects to `back` (I don't know exactly what is that). So initially you don't need to add anything in `app.ts` but I can take a look tomorrow.

Reply:
`redirect('back')` should send the user back to the referer of `/` (if no referer):
https://expressjs.com/en/4x/api.html#res.redirect

Reply:
FYI, i just tested locally (using google) and logout worked as expected